charts/jenkins/unittests/jcasc-config-test.yaml

```diff
@@ -1544,5 +1544,207 @@ tests:
                 usageStatisticsEnabled: true
             unclassified:
               location:
-                adminAddress: 
+                adminAddress:
                 url: http://RELEASE-NAME-jenkins:8080
+
+  - it: legacyRemotingSecurityEnabled = false (even though jenkins is 2.325 or older)
```

Comment:
what happens if the tag is not a semver tag and `legacyRemotingSecurityEnabled` is not set?

I can't easily tell from reading the code.

(would be good if there was a test showing what it does)

Reply:
Yeah, seems Helm does not have a nice way of handling this error or providing a nicer message... but adding that regex thing from #623 is also a bit overkill?

Reply:
can we just back out the semver thing.

Default this legacy flag to false, document that people on x version or older will need to enable this flag if they want to keep it enabled

Reply:
Sure, I don't mind doing that, but that means a breaking change and a major version bump?

Reply:
Cool, I'll open a separate PR once this one is merged, thanks